src/context/WalletProvider/KeepKey/components/RecoverySentenceEntry.tsx

```diff
@@ -43,6 +43,7 @@ export const KeepKeyRecoverySentenceEntry = () => {
   const [characterInputValues, setCharacterInputValues] = useState(
     Object.seal(new Array<string | undefined>(maxInputLength).fill(undefined)),
   )
+  const [passphrase, setPassphrase] = useState(new Array(0))
```

Comment:
- `new Array(size)` is generally discouraged in favor of the array literal notation except for very specific use cases, see:
https://eslint.org/docs/latest/rules/no-array-constructor
```suggestion
  const [passphrase, setPassphrase] = useState([])
```
- add a generic to `useState<T>`, passphrase currently has `any` type